core/templates/dev/head/services/ResourceService.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for Resources.
+ */
+
+oppia.factory('ResourceService', ['$http',
```

Comment:
Also, break after '[' if the contents span multiple lines. /cc @apb7 @lilithxxx

Reply:
Done.

Reply:
@seanlip There is an issue already open (#6047) to break after '(' if the content span multiple lines. Thanks!